src/containers/Dashboard/RecurringTransactions/RecurringTransactionForm.tsx

```diff
@@ -71,6 +76,12 @@ export const RecurringTransactionForm = ({
     currency,
     inputSats,
   }: FormValues) => {
+    if (isSendLoading) {
```

Comment:
here is the confirmation dialog, so it probably need to be here

or maybe is there a way how to disable the conf dialog buttons?